go.mod

```diff
@@ -89,6 +89,7 @@ require (
 	github.com/containerd/console v1.0.3 // indirect
 	github.com/containerd/containerd v1.6.2 // indirect
 	github.com/cosmos/btcutil v1.0.4 // indirect
+	github.com/cosmos/gogoproto v1.4.0 // indirect
```

Comment:
I believe this package might not be needed, I just checked "github.com/tendermint/flutter" dependencies and it has none, so this indirect dependency doesn't seem to be relevant for these changes.

Reply:
Ahh this must have been added by mistake

Reply:
done by [e60ee84](https://github.com/ignite-hq/cli/pull/2475/commits/e60ee847eaf399b99ae59a6ee0d40e170d4b7a0f).